lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4404,6 +4404,9 @@ namespace {
     AccessorDecl *tryCreateConstexprAccessor(const clang::VarDecl *clangVar,
                                              VarDecl *swiftVar) {
       assert(clangVar->isConstexpr());
```

Comment:
Can you add a comment below this line explaining what we're doing and why we're doing it? As of now, I don't fully understand _why_ this fixes the issue and it seems like a bit of a hack (which might be OK if this is just a stop-gap solution to prevent the crash, but if that's the case, please link an SR in the comment [I think the SR in question here would be the one for dependent types]).

Reply:
I added a comment. So I arrived at this fix by going through the stack trace. The crash was caused on clang side where it asserts on whether the expr has a dependent value. So I just tried mimicking it (Instead of letting it crash peacefully exit).

Reply:
This comment is good, but I'm not sure we've quite figured out _why_ this assertion is there. I assume clang doesn't have a similar branch + bail. 

Maybe you could use `isUsableInConstantExpressions` instead as the check. I think this will be a bit more self-explanatory, so the comment could just be (something along the lines of) "only import vars that are usable in a constant expression because otherwise we might fail to evaluate and find a value." 

What do you think?

Reply:
Ah yeah that makes a lot of sense, should have dug deeper I guess.